timetable/forms.py

```diff
@@ -160,3 +161,78 @@ def clean(self):
         #TODO check for encoding
         #TODO check for csv import errors
         return cleaned_data
+
+class SelectDateForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday)
+    
+    
+class AddDayPlanForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday())
+    i = 0
```

Comment:
The next 4-5 lines are equivalent to
```python3
enumerate(Schedule.objects.all())
```
Also not sure why `iterator()` was used. It probably would make database queries more expensive.